desktop/core/requirements.txt

```diff
@@ -72,3 +72,4 @@ tabulate==0.8.9
 thrift==0.13.0
 thrift-sasl==0.4.2
 git+https://github.com/gethue/django-babel.git
+git+https://github.com/agl29/django-mako.git
```

Comment:
Nice!

Forked to https://github.com/gethue/django-mako, so update it there?

Reply:
Done